### ISSUE TITLE
Firefox 57 support?

### ISSUE THREAD

**Issue by GiRaFa-SM** [wesley-chen/imagepicker#9]
Will you port this extension to be compatible with FF 57?

**Comment by FreshhhHub** [comment]
+1 same request! 👍

**Comment by john5698** [comment]
I do not think this Firefox extension is maintained. I have had success so far of looking at Image Picka, identifying the features that are missing and adding feature requests to https://github.com/eight04/image-picka as the developer seems very responsive
